packages/viewer/src/lib/stream/streamAttributes.ts

```diff
@@ -46,6 +53,16 @@ export const toProtoStreamAttributes = (
           ? { value: attributes.experimentalGhosting?.opacity }
           : attributes.experimentalGhosting?.opacity,
     },
+    featureLines: {
+      lineWidth: attributes.featureLines?.width,
+      lineColor: attributes.featureLines?.color
+        ? {
```

Comment:
Do we have a default color if only a width is defined?

Reply:
it defaults to black